coral-schema/src/main/java/com/linkedin/coral/schema/avro/SchemaUtilities.java

```diff
@@ -157,19 +157,25 @@ static Schema getCasePreservedSchemaFromTblProperties(@Nonnull final Table table
 
     // First try avro.schema.literal
     String schemaStr = readSchemaFromSchemaLiteral(table);
+    Schema schema = null;
```

Comment:
Actually this is just for simpler implementation, given we only have `schemaStr`, it's not clean to convert it to schema, apply the visitor and then convert it back to schemaStr for `dali.row.schema` case if we don't introduce `schema` variable here. But let me know if you have an idea to optimize it.

Reply:
Thanks for the explanation. Understood. My concerns is in the code
`    if (!Strings.isNullOrEmpty(schemaStr)) {
      LOG.info("Schema found for table {}", getCompleteName(table));
      LOG.debug("Schema is {}", schemaStr);
      return schema;
    }`
It checks and logs `schemaStr` but return `schema`. From the current implementation, it is the same thing. However, it may diverge due to a bug and make it harder to debug by looking at the log.

Reply:
Thanks for explaining it, I modified it to be `if (schema != null)` to avoid the potential divergence.